pachyderm/values.yaml

```diff
@@ -145,6 +145,22 @@ pachd:
     ssl: "disable"
     user: ""
     password: ""
+    cloudsqlAuthProxy:
+      enabled: true
+      image:
+        repository: "gcr.io/cloudsql-docker/gce-proxy"
+        tag: "1.23.0"
+      resources:
+        requests:
+          # The proxy's memory use scales linearly with the number of active
+          # connections. Fewer open connections will use less memory. Adjust
+          # this value based on your application's requirements.
+          memory: "2Gi"
```

Comment:
We generally haven't hardcoded requests and limits and left that as an exercise for the user depending on their cluster environment.